### ISSUE TITLE
Support chatting with a group of notes

### ISSUE THREAD

**Issue by wenlzhang** [exoascension/vault-chat#34]
It would be great if one can specify specific groups of notes to chat with, to avoid using too many tokens and to make the answers more specific. There can be different methods to determine the groups of notes to chat with.

Method 1: Chat with notes in a folder. For example, one can right-click on a folder and then select to chat with the folder.

Method 2: Chat with notes given by Dataview. With Dataview, one can query notes based on folders, tags and more. Then Vault Chat can chat with these notes given by a Dataview query. The following plugin also uses Dataview to query and operate on notes. Therefore, maybe some code can be used to achieve this:

- https://github.com/ivaneye/obsidian-files-cooker

**Comment by cpaika** [comment]
Yes, love this.  Method 1 is really easy to implement, but doing it with Dataview would be really powerful...will implement

**Comment by shamil900** [comment]
can crete plugin For mobile?
